modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -518,16 +518,18 @@ cv::gimpl::ie::GIEExecutable::GIEExecutable(const ade::Graph &g,
 void cv::gimpl::ie::GIEExecutable::run(cv::gimpl::GIslandExecutable::IInput  &in,
                                        cv::gimpl::GIslandExecutable::IOutput &out) {
     // General alghoritm:
-    //     1. Get input message from IInput
+    //     1. Since only single async request is supported
+    //        wait until it is over and start collecting new data.
     //     2. Collect island inputs/outputs.
     //     3. Create kernel context. (Every kernel has his own context.)
-    //     4. Since only single async request is supported
-    //        wait until it is over and run kernel.
-    //        (At this point, an asynchronous request will be started.)
-    //     5. Without waiting for the completion of the asynchronous request
+    //     4. Without waiting for the completion of the asynchronous request
```

Comment:
This text is hard to read. I'd rephrase it as "go to the next frame without ..."

Reply:
Fixed